src/toolbox/utils/array/contains.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { contains } from "./contains";
+import { expect } from "chai";
+import "mocha";
+
+describe("contains", () => {
+  const tests = [
```

Comment:
What about tests for other types of values that can be in an array?
null, undefined, true, false, {}, [] etc. ?